akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteReDeploymentSpec.scala

```diff
@@ -98,7 +98,7 @@ object RemoteReDeploymentMultiJvmSpec {
   class Echo(target: ActorRef) extends Actor with ActorLogging {
     def receive = {
       case msg ⇒
-        log.info(s"received $msg from ${sender()}")
+        log.warning(s"#18366 received $msg from ${sender()}")
```

Comment:
A little less cryptic way to include it perhaps? In case this would land in an actual release.
Like " likely relates to issue #..." etc?